calculator.js

```diff
@@ -0,0 +1,25 @@
+var calc = {
+  add: function (addend1, addend2) {
+    return addend1 + addend2;
+  },
+
+  subtract: function (sub1, sub2) {
+    return sub1 - sub2;
+  },
+
+  mutliply: function (multi1, multi2){
```

Comment:
One thing that might be tripping you up in the console: if you're trying to call `calc.multiply` instead of the "alternate" spelling here ;-)